howto/stream/debug.md

```diff
@@ -0,0 +1,36 @@
+Debugging issues that occur when streaming your application can be tricky. The following instructions give you some pointers on how to start tracking down streaming issues.
+
+First of all, check the error message that occurs. Unfortunately, it is often hard to determine why a connection fails, which is why the error message does not always give a clear indication of the source of the error. Typical errors are covered though, so in these cases, the error message should give you an idea on where to look.
+
+If the error message does not help, check the WebRTC debug information. You can find this information in your browser:
```

Comment:
We currently only support browser based clients so I think assuming the client is always browser based here is fine.